SimGeneral/TrackingAnalysis/python/trackingParticleNumberOfLayersProducer_cff.py

```diff
@@ -1,3 +1,6 @@
 from SimGeneral.TrackingAnalysis.trackingParticleNumberOfLayersProducer_cfi import *
 from Configuration.StandardSequences.Eras import eras
 eras.fastSim.toModify(trackingParticleNumberOfLayersProducer, simHits=['famosSimHits:TrackerHits'])
+eras.trackingPhase2PU140.toModify(trackingParticleNumberOfLayersProducer, simHits = cms.VInputTag(cms.InputTag("g4SimHits","TrackerHitsPixelBarrelLowTof"),
+                                                                                                  cms.InputTag("g4SimHits","TrackerHitsPixelEndcapLowTof"))
+)
```

Comment:
Could you use the same style I used above for fastSim, i.e.
```
simHits = ["g4SimHits:TrackerHitsPixelBarrelLowTof", "g4SimHits:TrackerHitsPixelEndcapLowTof"]
```
? (same also for `simHitTPAssociation_cfi.py`)

It is more compact to read and enables typo checking.